docs/kratos/concepts/credentials/username-email-password.mdx

```diff
@@ -410,6 +410,76 @@ $firescrypt$ln=<mem_cost>,r=<rounds>,p=<parallelization>$<salt>$<hash>$<salt_sep
 $firescrypt$ln=14,r=8,p=1$sPtDhWcd1MfdAw==$xbSou7FOl6mChCyzpCPIQ7tku7nsQMTFtyOZSXXd7tjBa4NtimOx7v42Gv2SfzPQu1oxM2/k4SsbOu73wlKe1A==$Bw==$YE0dO4bwD4JnJafh6lZZfkp1MtKzuKAXQcDCJNJNyeCHairWHKENOkbh3dzwaCdizzOspwr/FITUVlnOAwPKyw==
 ```
 
+### MD5
+
+Ory Identities doesn't hash password by MD5 but can compare stored MD5 hashes and migrate to configured hasher
+(`hashers.algorithm`). MD5 hashes can be configured in two different ways. Format is dependent on whether the hash is salted or
+not.
+
+#### Plain Format (for hashes without salt)
+
+```
+$md5$<hash>
+```
+
+#### Parameters
+
+- `hash`(`string`): The computed hash by the MD5 algorithm encoded in base64.
+
+#### Example
+
+```
+$md5$CY9rzUYh03PK3k6DJie09g==
+```
+
+#### Salted Format (for hashes with salt)
+
+If the salting should be performed before computing the MD5 hash, this format should be used.
+
+```
+$md5$pf=<salting-format>$<salt>$<hash>
+```
+
+#### Parameters
+
+- `salting-format`(`string`): Format string specifies how salting should be performed. It should be encoded in base64. For more
+  details, take a look at the [section below](#salting-format-parameter).
+- `salt`(`string`): A sequence of bits, known as a cryptographic salt encoded in base64.
+- `hash`(`string`): The computed hash by the MD5 algorithm encoded in base64. This method operates on two different stages. First,
+  it generates an input string to be given to MD5 algorithm with `salting-format`, `salt` and the User's password, in the second
+  stage generated string in the previous one is passed to the MD5 algorithm as an input. For more details, take a look at the
+  [section below](#salting-format-parameter).
```

Comment:
What do you mean when you're talking about "this method"? From what I understand, `hash` itself isn't a method. 
Does "this method" refer to the salted format in general? If so, please describe it (the two stages) directly under the `salted format (hashes with salt)` heading.

Reply:
Exactly, "this method" is refers to the "salted format". I'm going to move that description under the "salted format" heading as you say, thank  you :-)

Reply:
Sure :) LMK when you update this, I'll have another look.